README.md

```diff
@@ -8,7 +8,23 @@
 
 `pip install guidebook-api-python`
 
-# PyPI Pacakge Management
+# Example Usage
+
+```
+from guidebook import api_requestor
+
+api_key = 'example_key'
+
+api_client = api_requestor.APIRequestor(api_key)
+
+api_url = 'http://guidebook.com'
```

Comment:
we may want to specify an open api url that will return data so that people can plug in this code example with a useful result:

e.g.

`api_url = https://builder.guidebook.com/open-api/v1/sessions/`